File: 026. Remove Duplicates from Sorted Array/26. Remove Duplicates from Sorted Array.c
int removeDuplicates(int* nums, int numsSize) {
    int i = 0;
    for (int j = 1; j < numsSize; j++){
        if (nums[j] != nums[i]){
            nums[++i] = nums[j];
        }
    }
    return (numsSize <= 1 ? numsSize : i + 1);
}

File: 026. Remove Duplicates from Sorted Array/26. Remove Duplicates from Sorted Array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        lenn = len(nums)
        i = 0
        for j in range(1, lenn):    # refer to the key, two pointers
            if nums[j] != nums[i]:
                i += 1
                nums[i] = nums[j]
        return lenn if lenn <= 1 else i + 1
